gradle/verification-metadata.xml

```diff
@@ -56,6 +56,10 @@
       </trusted-artifacts>
       <ignored-keys>
          <ignored-key id="74dafdfd6dae2441" reason="Key couldn't be downloaded from any key server"/>
+         <ignored-key id="0a07e0e55e45f65b" reason="Used to sign jcifs - key couldn't be downloaded from any key server"/>
```

Comment:
Hmm... Are you not able to get this key locally? You should run with `--export-keys` to update the keyring to contain the new keys. If we ignore the keys, we should add the sha-sums instead.

Reply:
How to I get a key locally? I tried `gpg --recv-keys 0a07e0e55e45f65b` but got `gpg: keyserver receive failed: No name`.

Reply:
`/gradlew --write-verification-metadata pgp,sha256 help --export-keys`